Gemfile

```diff
@@ -19,7 +19,8 @@ gem 'jquery-rails'
 # Turbolinks makes following links in your web application faster. Read more: https://github.com/rails/turbolinks
 gem 'turbolinks'
 # Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
-gem 'jbuilder', '~> 2.0'
+# gem 'jbuilder', '~> 2.0'
+gem 'active_model_serializers', '~> 0.10.0'
```

Comment:
This gem is written to use in object oriented way which rails always follow OOPs concept compare to jbuilder
This gem is now part of rails 5.0 if we use rails 5 as API project.